integration_tests/dbt_project.yml

```diff
@@ -10,11 +10,18 @@ models:
 
 vars:
   hubspot_schema: hubspot_integration_tests_63
+  consistency_test_contact_list_exclude_fields: ## For hubspot__contact_lists consistency test. Can be removed after this coming release.
```

Comment:
Should we remove these fields prior to release?

Reply:
We can keep them in for now and then remove in the next update.